content/engine/reference/commandline/container_exec.md

```diff
@@ -9,6 +9,14 @@ aliases:
 layout: cli
 ---
 
+> **Introducing Docker Debug**
+>
+> Dive into Docker Debug, a replacement for debugging with `docker exec`. Access
```

Comment:
It prepend something like: "To easily get a debug shell into any container, we created a new command `docker debug`." And then proceed with "It's a replacement..."

-> put the problem that most people have when they search for docker exec at the beginning (which is getting a shell into their container)
-> tell them we have created a better tool (this is what you already wrote)

Reply:
Thanks, @joe0BAB. How about the wording below?

> To easily get a debug shell into any container, use `docker debug`.
> Docker Debug is a replacement for debugging with `docker exec`. With it, you
> can access any container or image, even slim ones, without modifications.
> Plus, you can bring along your favorite debugging tools in its customizable
> toolbox.

Reply:
Love this! Let's go with this!